storage-proofs-porep/src/stacked/vanilla/create_label/multi.rs

```diff
@@ -322,9 +322,9 @@ fn create_layer_labels(
 
             while producer_val < i {
                 if !printed {
-                    debug!("PRODUCER NOT READY! {}", i);
+                    trace!("PRODUCER NOT READY! {}", i);
```

Comment:
I also see this logged thousands of time per run.  I'd argue this individual log line could be removed.  Perhaps a map of the `i` value to specific `count` may be needed if this information is important and it could be logged at the end along with the total count?

Reply:
A map of `i` to `count` may be too large to be logged. I agree with remove this line.